src/askContainer/treeViews/treeViewProviders/helpViewProvider.ts

```diff
@@ -28,7 +28,7 @@ export class HelpViewProvider implements vscode.TreeDataProvider<PluginTreeItem<
         return element;
     }
 
-    getChildren(element?: PluginTreeItem<Resource>): PluginTreeItem<Resource>[] {
+    getChildren(element?: PluginTreeItem<Resource>): Array<PluginTreeItem<Resource>> {
```

Comment:
nit: Trying to understand the change here. Is this another style change , since the [generics handbook](https://www.typescriptlang.org/docs/handbook/generics.html#working-with-generic-type-variables) mentions TS works fine with either syntax?

Reply:
Yeah, it just another style change. As in our lint config, we choose the array-simple option: https://github.com/typescript-eslint/typescript-eslint/blob/v3.10.1/packages/eslint-plugin/docs/rules/array-type.md#array-simple. It requires us to use T[] for simple types but Array<T> for all other types (union types, intersection types, object types, function types, etc)